NOTICE

```diff
@@ -1,2 +1,7 @@
 Firecracker
-Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We also seem to have a lot of files that state 2018/2019 copyright dates, but have been updated in 2020. Should we update those too?

Reply:
That is a different task and we should open an issue on that.